gramex/ml_api.py

```diff
@@ -203,8 +207,12 @@ class ModelStore(cache.JSONStore):
 
     def __init__(self, path, *args, **kwargs):
         _mkdir(path)
-        self.data_store = op.join(path, "data.h5")
-        self.model_path = op.join(path, op.basename(path) + ".pkl")
+        if op.exists(op.join(path, "data.h5")):
```

Comment:
Why is this condition added? The `self.data_store` and `self.model_path` variables are declared here to let the model know the paths to files where training data and the model (respectively) would be stored. They don't need to exist at the beginning.